client/lib/olark/index.js

```diff
@@ -164,6 +168,24 @@ const olark = {
 		}
 	},
 
+	showAvailabilityNotice() {
+		const { details, isOperatorAvailable } = olarkStore.get();
+		const noticeOptions = {
+			duration: 10000,
+			showDismiss: false
+		};
+
+		if ( ! ( details.isConversing || window.location.pathname === '/help/contact' ) ) {
```

Comment:
`window.location` being here might raise a few eyebrows but its here because I was trying to avoid duplication in the `HelpContact` component.